lib/Controller/Widget.php

```diff
@@ -457,13 +465,31 @@ public function editWidget(Request $request, Response $response, $id)
         }
 
         $module = $this->moduleFactory->getByType($widget->type);
+        $widgetName = $widget->getOptionValue('name', $module->name);
```

Comment:
If we set this on edit, it means that when the user changes the name of an item in the library, it will not also change on the playlist editor (because the old item will be set here).

So, I don't think we need to do anything on edit?

Reply:
@dasgarner The `widgetName` comes out to be still empty if I remove these changes here on edit.

Reply:
Okay, I think I got it now. I'll update the PR